crds/jwst/specs/fgs_apcorr.rmap

```diff
@@ -0,0 +1,73 @@
+header = {
+    'classes' : ('Match', 'UseAfter'),
+    'derived_from' : 'Hand made 2019-10-22',
+    'file_ext' : '.fits',
+    'filekind' : 'APCORR',
+    'filetype' : 'APCORR',
+    'instrument' : 'FGS',
+    'mapping' : 'REFERENCE',
+    'name' : 'jwst_fgs_apcorr.rmap',
```

Comment:
Do you happen to know if there are any implications to this name field not matching the actual filename?  Looking through the old specs, some match and some don't.

Reply:
No, I do not, but I'll make match.

Reply:
As CRDS renames files,  it copies "name" to "dervived_from" and sets "name" to match the generated name.  derived_from is used to easily diff an rmap against its parent.

If you are doing a reference file submission  with automatic rmap generation,  in the final rmap added to CRDS it will reflect the baseline rmap for the update which in turn is related to the context selected on the submission form.   

If you are doing a manual rmap update (download-modify-upload-with-auto-renaming) it should reflect the name of the rmap you downloaded and hence the baseline for the change.  Ideally,  let CRDS do all renaming.

There is a boundary issue so the first .rmap effectively has no parent.  (Strictly speaking,  it is the spec file which is not in CRDS content.)  In this case the main consequence is that  diffs against a parent aren't supported.